.ci_support/win_c_compilervs2015vc14.yaml

```diff
@@ -0,0 +1,22 @@
+c_compiler:
+- vs2015
+channel_sources:
+- conda-forge,defaults
+channel_targets:
+- conda-forge main
+jpeg:
+- '9'
+libtiff:
+- 4.1.0
```

Comment:
I'd prefer the first version to be built against libtiff 4.0.10 because of https://github.com/python-pillow/Pillow/issues/4237

Reply:
I think you **should** be able to use it with libtiff 4.0, but maybe lets just patch up libtiff 4.1?

Reply:
https://github.com/conda-forge/libtiff-feedstock/pull/49

Reply:
Works for me too